operations/mimir-mixin/alerts/alerts.libsonnet

```
@@ -244,6 +244,23 @@ local utils = import 'mixin-utils/utils.libsonnet';
             ||| % $._config,
           },
         },
+        {
+          // Alert if an ingester instance has no tenants assigned while other instances in the same cluster do.
+          alert: $.alertName('IngesterInstanceHasNoTenants'),
+          'for': '1h',
+          expr: |||
+            (min by(%(alert_aggregation_labels)s, %(per_instance_label)s) (cortex_ingester_memory_users) == 0)
+            and on (%(alert_aggregation_labels)s)
+            # Only if any ingester replica in the cluster has a tenant assigned
```

Unfortunately we can't use the word "cluster" otherwise a linter check fails:

```suggestion
            # Only if there's a least 1 tenant in Mimir.
```

---

Thanks, changed as per your suggestion 👍